sdk/textanalytics/azure-ai-textanalytics/samples/sample_analyze.py

```diff
@@ -0,0 +1,151 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_analyze_text.py
+
+DESCRIPTION:
+    This sample demonstrates how to submit a collection of text documents for analysis, which consists of a variety
+    of text analysis tasks, such as Entity Recognition, PII Entity Recognition, Entity Linking, Sentiment Analysis,
+    or Key Phrase Extraction.  The response will contain results from each of the individual tasks specified in the request.
+
+USAGE:
+    python sample_analyze_text.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_TEXT_ANALYTICS_ENDPOINT - the endpoint to your Cognitive Services resource.
+    2) AZURE_TEXT_ANALYTICS_KEY - your Text Analytics subscription key
+"""
+
+
+import os
+
+
+class AnalyzeSample(object):
+
+    def analyze(self):
+        # [START analyze]
+        from azure.core.credentials import AzureKeyCredential
+        from azure.ai.textanalytics import TextAnalyticsClient, \
+            EntitiesRecognitionTask, \
+            PiiEntitiesRecognitionTask, \
+            EntityLinkingTask, \
+            KeyPhraseExtractionTask, \
+            SentimentAnalysisTask
+
+        endpoint = os.environ["AZURE_TEXT_ANALYTICS_ENDPOINT"]
+        key = os.environ["AZURE_TEXT_ANALYTICS_KEY"]
+
+        text_analytics_client = TextAnalyticsClient(
+            endpoint=endpoint, 
+            credential=AzureKeyCredential(key),
+            api_version="v3.2-preview.1"
+        )
+
+        documents = [
+            "I had a wonderful trip to Seattle last week.",
+            "I'm flying to NYC tomorrow. See you there."
+        ]
+
+        poller = text_analytics_client.begin_analyze(
```

Comment:
same comment here. 
Is it common for a user to always call all tasks? or are there combinations that are more common?
what can the user do by having multiple results from one set of documents?
an idea could be to show the before and after analyze.